docs/inference-providers/providers/sambanova.md

```diff
@@ -0,0 +1,18 @@
+# SambaNova
+
+![https://sambanova.ai/](https://sambanova.ai/hubfs/sambanova-logo-black.png)
```

Comment:
maybe @ChunTeLee can generate 1 nice visual for each provider (cc @gary149)

let me know! (can be in a later PR)